src/utils/injectReducer.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import hoistNonReactStatics from 'hoist-non-react-statics';
+import { ReactReduxContext } from 'react-redux';
 
 import getInjectors from './reducerInjectors';
```

Comment:
I think it's a matter of preference. When exporting a functional component, I would say yes, because having a default export without an explicit function name will show nameless components in snapshots and shallow renders. In this case, it's just a function that is exported and it doesn't need to be references by its name.